### PR TITLE
Add docs for non-AD desktop access for OSS users

### PR DESCRIPTION
**Pull request by probakowski**

### CODE REVIEW

File: docs/pages/desktop-access/getting-started.mdx [hunk 1]
```diff
@@ -8,21 +8,20 @@ This guide will help you configure Teleport to provide secure, passwordless acce
 to Windows desktops. This configuration does not require an Active Directory domain.
 
 <Admonition type="note">
-Passwordless access for local users is an Enterprise-only feature.
+Passwordless access for local users is limited to 5 desktops in open source Teleport.
```
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:11]:
```suggestion
Passwordless access for local users is limited to 5 desktops Teleport Community Edition.
```

File: docs/pages/desktop-access/getting-started.mdx [hunk 2]
```diff
@@ -8,21 +8,20 @@ This guide will help you configure Teleport to provide secure, passwordless acce
...
 
-For open source Teleport, consider integrating Teleport with Active Directory
-for automatic discovery by reading [Desktop Access with Active Directory](./active-directory.mdx).
+If you have more than 5 desktops configured using method below in open source Teleport
+you won't be able to connect to any of them.
```
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:14]:
```suggestion
If you have more than 5 desktops registered using the method below, Teleport
Community Edition will not allow connections to any of them.
```

File: docs/pages/desktop-access/getting-started.mdx [hunk 3]
```diff
@@ -8,21 +8,20 @@ This guide will help you configure Teleport to provide secure, passwordless acce
...
 
-Teleport Enterprise users can also mix the Teleport Active Directory integration
-with the static host definitions described below.
+Teleport Enterprise users can have unlimited number of desktops.
```
Comment (ptgott) [inline docs/pages/desktop-access/getting-started.mdx:16]:
```suggestion
Teleport Enterprise users can enroll an unlimited number of desktops.
```

**Review by ptgott — APPROVED** [review]
Approved assuming Zac's feedback is incorporated

File: docs/pages/desktop-access/getting-started.mdx [hunk 4]
```diff
@@ -8,21 +8,20 @@ This guide will help you configure Teleport to provide secure, passwordless acce
 to Windows desktops. This configuration does not require an Active Directory domain.
 
 <Admonition type="note">
-Passwordless access for local users is an Enterprise-only feature.
+Passwordless access for local users is limited to 5 desktops Teleport Community Edition.
```
Comment (zmb3) [inline docs/pages/desktop-access/getting-started.mdx:11]:
```suggestion
Passwordless access for local users is limited to 5 desktops in Teleport Community Edition.
```

**Review by zmb3 — APPROVED** [review]

**Review by lsgunn-teleport — APPROVED** [review]
> Well technically community can enroll an unlimited number too. The enforcement happens when you try to connect, not at enrollment time.

I'm not sure if adding this is necessary, but maybe for consistency with Paul's comment:
If you attempt to enroll more than five desktops by listing them under `non_ad_hosts`, Teleport Community Edition won't allow connections to any of them.

Should we use hosts in all three places?
If you attempt to enroll more than five hosts...
... an unlimited number of hosts.

**Pull request merged**